ansible/roles/software/ckan/catalog/ckan-app/templates/etc_apache2_sites-enabled_ckan.conf.j2

```diff
@@ -72,11 +72,16 @@ SSLEngine on
     RewriteCond %{HTTP_REFERER}    !{{saml2_idp_entry}} [NC]
     RewriteCond %{HTTP_COOKIE}     !auth_tkt
     RewriteRule ^(.*)$ {{ url_readonly }}$1 [R,L]
+
 {% elif catalog_ckan_readwrite_configuration == 'readonly' %}
     RewriteRule   "^(/en)?/user/login"  "{{ url_writable }}/user/login" [R,L]
     RewriteRule   "^(/en)?/user/_logout"  "{{ url_writable }}/user/_logout"  [R,L]
     RewriteRule   "^(/en)?/user/logged_in"  "{{ url_writable }}/user/logged_in"  [R,L]
     RewriteRule   "^(/en)?/login_generic"  "{{ url_writable }}/login_generic"  [R,L]
+
+    RewriteCond %{HTTP_REFERER} !{{saml2_idp_entry}} [NC]	
+    RewriteRule ^(\/?)$ /dataset [R]
```

Comment:
I'm not really sure how this solves the issue. The problem is that `https://catalog.data.gov/` is redirecting to `http://catalog.data.gov/dataset`, so this new rule doesn't change that.

Reply:
@adborden, we are now redirecting to `/dataset` at CKAN level [inside DataGovTheme](https://github.com/GSA/ckanext-datagovtheme/blob/master/ckanext/datagovtheme/plugin.py#L81:L81).
Issue #2083 mention we have two redirections and we expect just one. 
I assume we have the _https_ redirection in Apache and the second one, inside CKAN.
I think we need to add this redirection inside apache like we already [are doing at the inventory](https://github.com/GSA/datagov-deploy/blob/2083_redirect2/ansible/roles/software/ckan/inventory/templates/etc/apache2/sites-enabled/ckan.conf#L45:L45)